### DISCUSSION TITLE
AR眼鏡「透視」病人身體，降低微創手術風險

### DISCUSSION THREAD

**Discussion by BlahFish1006** [1101-GS4719-C/Python-Community-GS4719-C#1022] [科技新知]
本篇新聞在講一個花蓮慈濟醫院與台灣骨王生技公司合作共同研究，超越微創的脊椎手術！
透過AR擴增實境的技術，利用攝影機與影像分析來計算3度空間的位置，讓顯示器中顯示虛擬物件，並與真實世界進行實際互動，
讓醫師在手術中可達到用肉眼直觀顯示，隱藏在皮膚肌肉下的神經與血管，讓器械在小於1mm的誤差下，能精準的投射出真實的體內結構，降低脊椎手術時可能傷害周圍的神經與血管的風險。

🔆而目前該研究在手術術前計畫的合作結果，也已實際應用在臨床上，成功幫助0402台鐵太魯閣號出軌事件中，一名嚴重骨盆骨折傷者順利重建骨盆。

不久前跟朋友去逛了去年(2022)的台灣醫療科技展，發現近期有越來越多混合實境的技術被實際應用於臨床上，協助醫師進行手術、診治、甚至學習，不斷突破人們對醫病體驗中的想像，甚是令人興奮。

新聞來源：https://www.tcnews.com.tw/medicine/item/16679.html

**Comment by delia07152** [comment]
好酷，感覺這項技術成熟後，不只可以應用在脊椎手術，但因為我沒甚麼概念，所以不確定誤差小於1mm算不算精準?

**Comment by Sylvie0601** [comment]
我也覺得超級酷的ㄟ!我覺得技術成熟感覺不只可以運用在骨科，感覺大腦、心臟這種超級難手術的地方也可以運用，甚至視神經手術、癌症手術應該也都可以，讓AR跟達文西手臂這些醫療器具結合尋找腫瘤這種，感覺會大大降低手術風險!

**Comment by Tuoresa** [comment]
好讚的一項技術！
感覺能非常有效的提升手術的精準度與效率！

過去也見識過很多科技為醫療帶來極大的幫助的實例
能夠大幅提升醫病關係建立及就醫體驗
是非常令人期待往後發展的一個領域呢

**Comment by Valentinaa0214** [comment]
很酷的技術應用，
既可以降低手術風險，甚至可能可以降低手術操作時間，
對醫生和病人來說都是好事！

**Comment by Kotori0501** [comment]
這項技術真的很好，大幅降低病人手術後的傷害風險，而且又可以更加仔細的進行手術，對大家來說都是一件好事，而且這部分的內容感覺AI很適合做輔助的角色來幫助醫生進行手術，不知道之後會不會由AI來完全主導手術

**Comment by dabbie109303501** [comment]
這項技術真的很棒!
AR不只能拿來玩遊戲 還能拿來做如此有意義造福大眾的事
希望能盡早研發透徹並推廣給醫生做練習 增加各式手術的成功率!

**Comment by Guagua1234** [comment]
這項技術感覺會大幅度降低醫生的壓力及手術時間，
曾經看過新聞上醫生的平均休假日期及排班狀況，
在從事醫生這個職業難免會遇到生離死別，
而著個壓力也會反饋回去醫生身上，
如果可以降低風險，那必定可以降低醫生的壓力使他們手術時間變短，
希望這項技術可以盡早普及，使台灣的醫療資源及環境更加完善

**Comment by wenchi9023** [comment]
微創手術已經是許多人解決病痛問題時的一個選擇，而這項技術更是降低病人和醫生手術的風險，令人振奮的消息!
希望這項技術能趕快更加純熟，並運用在更廣泛醫療上

**Comment by Jasper525** [comment]
“良醫墨菲”中有出現過醫生在手術前用vr頭盔與手把模擬幾次手術流程操作，這也是非常好的應用，感覺可以跟手術中的ar利用相輔相成。

**Comment by harrylee1971** [comment]
這種技術是先建立在有良好的MRI和CT影像中，才可以擁有好的模擬圖片出來

**Comment by a45320614** [comment]
真的希望這項技術能夠發展成熟並且普及化來幫助更多有需要的人，而且這項技術不只能應用在臨床上，對於醫學生們應該也是個非常好的學習工具，透過模擬手術過程可以達到的學習效果與以往只能背誦書本上的知識會有巨大的差異，這也是教育可以利用科技成長的一個具體且明確的方向。

**Comment by HsuChen88** [comment]
輔助性的儀器加入AI後，大幅提升了醫療品質。

**Comment by Jzd314** [comment]
如果這項技術能夠繼續成熟配合，手術的成功率將會大大提升

**Comment by 110302550** [comment]
現在醫療技術一直在進步，但病毒也越來越厲害，所以我們也不能落下，相信醫良技術越來越優秀我們就能活的更加舒適，不會常常受到疾病的折磨

**Comment by ghost** [comment]
我覺得很好!我覺得技術成熟感覺不只可以運用在骨科，感覺大腦、心臟等高難手術的地方，可能可以會大大降低手術風險!

**Comment by ghost** [comment]
方便的同時也要小心其弊端，如果顯示位置有所偏移，導致醫師切斷動脈，將可能造成不可挽回的後果
